README.md

```diff
@@ -156,6 +156,18 @@ options:
 Each worksheet has an entry `Test Active` which can be either true or false.  If set to false, the worksheet will be skipped, unless it is
 the named worksheet using the `-w` command line switch.
 
+## Integration tests (cucumber)
+
+The `features` folder holds two sample cucumber tests that are a POC for a future project to replace the above spreadsheet-based ispec tests.
```

Comment:
should we remove reference to POC and state it is to run alongside the ispec tests for now, testing different features.

Reply:
Good point!